patches/tModLoader/Terraria/ModLoader/Core/LoaderUtils.cs

```diff
@@ -246,7 +246,7 @@ public static void MustOverrideTogether<T>(T t, params Expression<Func<T, Delega
 	{
 		int c = methods.Count(m => HasOverride(t, m));
 		if (c > 0 && c < methods.Length)
-			throw new Exception($"{t!.GetType()} must override all of ({string.Join('/', methods.Select(m => m.Name))}) or none");
+			throw new Exception($"{t!.GetType()} must override all of ({string.Join('/', methods.Select(m => m.ToOverrideQuery().Method.Name))}) or none");
```

Comment:
This is perfectly fine, but it's simpler to use `.ToMethodInfo()`.

See the implementation of `.ToOverrideQuery()` for details.